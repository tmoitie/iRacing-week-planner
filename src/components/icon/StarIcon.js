// @flow

import React from 'react';
import styles from '../../styles/main.module.scss';

export default function StarIcon() {
  return <span data-testid="StarIcon" className={`${styles.glyphicon} ${styles['glyphicon-star']}`} />;
}
