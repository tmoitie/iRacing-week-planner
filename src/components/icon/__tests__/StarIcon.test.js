import React from 'react';
import { render } from '@testing-library/react';
import { describe, test } from '@jest/globals';
import StarIcon from '../StarIcon';

describe('components/icon/StarIcon', () => {
  test('renders correctly', () => {
    const { container } = render(<StarIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
