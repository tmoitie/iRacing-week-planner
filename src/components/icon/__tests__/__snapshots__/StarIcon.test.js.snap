// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/icon/StarIcon renders correctly 1`] = `
<span
  class="glyphicon glyphicon-star"
  data-testid="StarIcon"
/>
`;
